### ISSUE TITLE
[Bug Styles]: Cart checkout. Background is not make black

### ISSUE THREAD

**Issue by vitalics** [magento-research/pwa-studio#598]
## This issue is for the following packages:

- [X] `venia-concept`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [X] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### Environment

| Question                    |    Answer     |
| --------------------------- | :-----------: |
| Magento version             |  |
| Operating System + version  |  |
| node.js version (`node -v`) |  |
| npm version (`npm -v`)      |  |

## Description
actual resusult when client form with infomation is opened(backound not change into black with opacity): 
![image](https://user-images.githubusercontent.com/8816260/49501542-29797f80-f884-11e8-9906-5acea16f52ef.png)

## Additional information:
backround color: black
opacity: 0.5

## Expected result:
background must look as you may see on the provided mockup
![image](https://user-images.githubusercontent.com/8816260/49501430-de5f6c80-f883-11e8-8b61-1846c9f5c07e.png)

## Possible solutions:
``` css 
.miniCart-footer {
margin: 0 0 /* was 0 1.5rem */
}
```
removing margin is nesessary or we would see background on footer form too(see screen below):
![image](https://user-images.githubusercontent.com/8816260/49503069-9fcbb100-f887-11e8-87b6-f01e454e8af7.png)

create background element(div)and set him position with next style rules:
``` css
.miniCart-background {
background-color: black;
opacity: 0.5;
/* position rules*/
}
``` 

## Actions:
- approve from architect(@zetlen )
- approve from product perspective: @ericerway or @awilcoxa

**Comment by vitalics** [comment]
Update.
- inside `.miniCart-body` add `<button class="mask-root" />` 
- `miniCart-footer` add `z-index: 2` and remove margins and paddings
- `form-footer flow-footer` remove margin
DOM:
![image](https://user-images.githubusercontent.com/8816260/49805608-9306fb80-fd66-11e8-8275-cccace9999b7.png)
RESULT:
![image](https://user-images.githubusercontent.com/8816260/49805630-a1edae00-fd66-11e8-8156-b8613289a597.png)

**Comment by zetlen** [comment]
Thanks, @vitalics, for an excellent summary of this issue. Potentially fixed by #717.

**Comment by awilcoxa** [comment]
Duplicate of #696  closing issue.
